### PR TITLE
LPS-173894 As a power user, I want to set notification configurations

### PR DESCRIPTION
**Pull request by devpedrotavares**
Related tickets:
- https://issues.liferay.com/browse/LPS-175117
- https://issues.liferay.com/browse/LPS-175121
- https://issues.liferay.com/browse/LPS-175123
- https://issues.liferay.com/browse/LPS-175427

### CODE REVIEW

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
ci:test:object

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:test:object

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java [hunk 1]
```diff
@@ -134,6 +137,40 @@ public String getTypeLanguageKey() {
...
+				notificationQueueEntryLocalService.getNotificationEntries(
+					type, status)) {
+
+			if (status != NotificationQueueEntryConstants.STATUS_UNSENT) {
+				notificationQueueEntryLocalService.updateStatus(
+					notificationQueueEntry.getNotificationQueueEntryId(),
+					NotificationQueueEntryConstants.STATUS_UNSENT);
+			}
+
+			_sendEmail(notificationQueueEntry);
```
[the remark this answers is not in the document]
Comment (devpedrotavares) [reply modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java]: Yes! I just replaced it.

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:test:object

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by evanilson-santana** [comment]
Hey @devpedrotavares I'm going to run the Object suit again since that last execution was before the current commits

cc: @marcelabc

**Comment by evanilson-santana** [comment]
ci:test:object

**Comment by evanilson-santana** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
> Hey @devpedrotavares I'm going to run the Object suite again since that last execution was before the current commits
> 
> cc: @marcelabc

Hey @evanilson-santana, in the last force-push, all I did was rebase on master.

**Comment by evanilson-santana** [comment]
hey @devpedrotavares can you please resolve these conflicts?

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
The ci:test:sf is failing because of a missing feature flag that belongs to another story. It will likely be fixed soon.

**Comment by devpedrotavares** [comment]
ci:test:sf

**Comment by devpedrotavares** [comment]
Forwarding directly: https://github.com/brianchandotcom/liferay-portal/pull/131100

**Pull request closed without merging**